webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -2,6 +2,8 @@ import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import Skeleton from 'react-loading-skeleton';
+import ArrowIcon from '@patternfly/react-icons/dist/js/icons/arrow-icon';
```

Comment:
This broke RPM builds:

```
ModuleNotFoundError: Module not found: Error: Can't resolve '@patternfly/react-icons/dist/js/icons/arrow-icon' in '/builddir/build/BUILD/foreman-2.4.0-develop/webpack/assets/javascripts/react_app/components/HostDetails/Audits'
```

Do we need a specific version for that to work?

Reply:
We do have rather an old version of this package in our packaging, but I can't find any clue on this package versioning and thus what version we need. @amirfefer ?

Reply:
The file `arrow-icon.js` exists even in an old version like `3.15.16`
We import patternfly 4 icons without any error
i.e
https://github.com/theforeman/foreman/blob/c4559edf0802a10ced3cdcee8620718e438383ba/webpack/assets/javascripts/react_app/components/Layout/components/CustomContextSelector.js#L12

Reply:
But never from dist. Is that an issue?

Reply:
@evgeni  This import is based on patternfly usage suggesting (for tree shaking) - https://github.com/patternfly/patternfly-react/tree/master/packages/react-icons

Hopefully, this PR fixes the issue https://github.com/theforeman/foreman/pull/8190

Reply:
Yeah, not sure what makes it exactly fail here. The linked PR looks like Worth a try, thanks!